database/models/messages.db.php

```diff
@@ -8,14 +8,15 @@
 use Doctrine\ORM\Mapping\ManyToOne;
 use Doctrine\ORM\Mapping\OneToMany;
 use Doctrine\ORM\Mapping\Table;
+use Doctrine\ORM\Mapping\OneToOne;
 
 #[Entity, Table(name: 'messages')]
 class Message
 {
     #[Id, Column, GeneratedValue]
     public int|null $id = null;
 
-    #[Column]
+    #[Column(type: "text")]
```

Comment:
Type text is the default I believe, so you don't have to specify it

Reply:
I checked, and this is changing the column type to `CLOB`, which is actually the same as `TEXT` in sqlite, as there is no length restriction by default in sqlite :) But this is why a long migration was generated, changing the message table.